src/resourcemgr/tpl/fileSelect.tpl

```diff
@@ -25,7 +25,7 @@
                                         <a href="{{downloadUrl}}" download="{{name}}" target="_blank" class="tlb-button-off download" title="{{__ 'Download this file'}}"><span class="icon-download"></span></a>
                                     {{/if}}
                                     {{#if permissions.delete }}
-                                        <a href="#" class="tlb-button-off" title="{{__ 'Remove this file'}}" data-delete=":parent li"><span class="icon-bin"></span></a>
```

Comment:
Just a remark here, if we checked that `data-delete` is not used elsewhere in the code based

Reply:
This is a "magic" statement, inherited from the dark age of TAO: 
- https://github.com/oat-sa/tao-core/blob/master/views/js/uikitLoader.js#L72
- https://github.com/oat-sa/tao-core-ui-fe/blob/master/src/deleter.js#L258
- https://github.com/oat-sa/tao-core-sdk-fe/blob/master/src/core/dataattrhandler.js#L70

We need to remove it otherwise, the deleted will still target it and trigger the global handler for deletion